modules/persistent-storage-csi-vol-populator-procedure.adoc

```diff
@@ -0,0 +1,137 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-vol-populators.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-vol-populator-procedure_{context}"]
+= Creating prepopulated volumes using volume populators
+
+The following procedure explains how to create a prepopulated persistent volume claim (PVC)  using the example `hellos.hello.example.com` Custom Resource Definition (CRD) created previously. In this example, rather than creating an actual data source, you are creating a file called "example.txt" that contains the string "Hello, world!" in the root directory of the volume.
+
+.Prerequisites
+
+* You are logged in to a running {product-title} cluster.
+
+* There is an existing custom resource definition (CRD) for volume populators.
+
+.Procedure
+
+. Create an instance of the `hello, world` CRD with the text "Hello, World!" by running the following command:
```

Comment:
```suggestion
. Create a Custom Resource (CR) instance of the `Hello` CRD with the text "Hello, World!" passed in as `fileContents` parameter by running the following command:
```

- CRD is called `Hello`, created in previous step
- now we're creating custom resource that has to conform to the CRD (which is kind of a schema)

Reply:
Fixed.